functions/streamtoidv/main.py

```diff
@@ -1,13 +1,58 @@
 import base64
 import json
+import logging
+import utils
 
-print('Loading function')
-
+from cis import validation, encryption
 
 def handle(event, context):
-    print("Received event: " + json.dumps(event, indent=2))
+
+    # Initialize Stream Logger
+    # Log level can be environment driven later in development.
+    log_level = logging.INFO
+    utils.set_stream_logger(level=log_level)
+    logger = logging.getLogger('cis-validator')
+    logger.info("Stream Processor initialized.")
+
     for record in event['Records']:
         # Kinesis data is base64 encoded so decode here
-        payload = base64.b64decode(record['kinesis']['data'])
-        print("Decoded payload: " + payload)
+        logger.info("Record is loaded.")
```

Comment:
@gdestuynder suggest that we go with user_id since that's the primary key for dynamo it's the only thing guaranteed.  Maybe once I have the scan code we could log the user_id and whether it's new user or an update.